framework/deproxy_server.py

```diff
@@ -17,10 +17,11 @@
 
```

Comment:
If you implement the chunked socket transmission for the deproxy server, as you did it for the client, then you can write real functional tests for https://github.com/tempesta-tech/tempesta/issues/1535 . So please go ahead and implement the functionality for the server side as well. This will close https://github.com/tempesta-tech/tempesta-test/issues/115

Reply:
What about TLS transmissions?

Reply:
Yeah, the things are little more complicated the. I believe we need to make `setsockopt(SO_SNDBUF)` for the desired chunk size. I'd propose to do this in a generator wrapper, which generates the sequence of chunk sizes (i.e. pass the socket as a parameter to the generator). The wrapper must remember the last value of chunk produced by the generator and make the call only if the new chunk differ. Some thing like this pseudocode:
```Python
# seq_gen() is provided by the caller and just emits sequence of integers.
def set_trx_chunk(sock, seq_gen):
   last_sz = 0
   while True:
        sz = seq_gen()
        if last_sz and sz == last_sz:
            setsockopt(sock, SO_SNDBUF, sz)
            last_sz = sz
```
Make sure with tcpdump or wireshark that this works.